packages/aragon-api/src/index.js

```diff
@@ -185,6 +229,21 @@ export class AppProxy {
     return value
   }
 
+  /**
+   * Get a value in from the application cache.
+   *
+   * @param  {string} key   The cache key to set a value for
+   * @return {Observable} An [RxJS observable](http://reactivex.io/rxjs/class/es6/Observable.js~Observable.html) that emits the cached value every time it changes. The type of the emitted values is cache key specific.
+   */
+  getCache (key) {
+    return this.rpc.sendAndObserveResponse(
+      'cache',
+      ['get', key]
+    ).pipe(
+      pluck('result')
+    )
+  }
```

Comment:
I had that in mind, but that breaks the API because `state()` is an subscription and `getCache` returns the first value and completes the observable.

Reply:
Perhaps then we could rewrite `getCache()` to use `state()` (pipe in `first()`)?